org.sebicodes.Prt.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.sebicodes.Prt
```

Comment:
`sebicodes.org` doesn't exist. This app-id doesn't meet the requirements.

Reply:
But I don't have any domain, what should i do?

Reply:
use one associated to the code hosting https://docs.flatpak.org/en/latest/conventions.html#application-ids